### PR TITLE
installer/windows: checkbox for opening SNOboard after installation

### PR DESCRIPTION
**Pull request by Fadila82**
What: checkbox for opening the SNOboard when the Finish button is clicked.
Acceptance criteria:
- The ExitDialog display a checkbox offering to open the SNOboard
- The checkbox is check by default, but can be unchecked by the user
- If the checkbox is checked, the SNOboard (http://localhost:14002/) will open in the default browser when the user clicks the Finish button

Why: So the user can directly access their dashboard after installation

Please describe the tests: no test
 
Please describe the performance impact: none

## Code Review Checklist (to be filled out by reviewer)
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

File: installer/windows/Product.wxs [hunk 1]
```diff
@@ -83,19 +83,26 @@
     <Property Id="STORJ_STORAGEDIR">STORAGEDIR</Property>
     <Property Id="STORJ_STORAGE">1.0</Property>
     <Property Id="STORJ_BANDWIDTH">2.0</Property>
+    <Property Id="WIXUI_EXITDIALOGOPTIONALCHECKBOXTEXT" Value="Launch SNOBoard" />
```
Comment (kaloyan-raev) [inline installer/windows/Product.wxs]: Let's change the text to "Open dashboard". We call it "SNOboard" internally, but users who just install the product most probably won't be familiar with this term.

File: installer/windows/Product.wxs [hunk 2]
```diff
@@ -83,19 +83,26 @@
...
+    <Property Id="WIXUI_EXITDIALOGOPTIONALCHECKBOX" Value="1"/>
```
Comment (mniewrzal) [inline installer/windows/Product.wxs]: Can we use more meaningful name for that? e.g. `WIXUI_EXITDIALOGOPENDASHBOARDCHECKBOX`

Comment (kaloyan-raev) [reply installer/windows/Product.wxs]: It's a built-in property in the standard ExitDialog of WiX Toolset :)

Comment (mniewrzal) [reply installer/windows/Product.wxs]: oh, then looks good ;)

**Pull request merged**